### PR TITLE
YAML batch 2 assets update

### PR DESCRIPTION
**Pull request by FedericoRondoni**

### CODE REVIEW

**Comment by FedericoRondoni** [comment]
Hi @santilland and @lubojr this batch is ready to be merged into the main branch. Once you have done the checks we can also proceed to tag Anca into the PR of the YAML batch_2.
Thanks!

**Comment by santilland** [comment]
We are still evaluating how to handle the "aggregated" collections, not sure how to proceed with the follow on batches. If we introduce a higher level catalog on top of the collections we might need to evaluate what information is on that level and what information is on the lower collection level.
I will keep you posted.

File: collections/E10a3_agricultural_production_area_change/E10a3_agricultural_production_area_change.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
...
 
-Earth Observation can provide timely information for monitoring border closures & transport restrictions and changing food demand on agricultural activities related to food production. 
-Satellite observations, such as from Sentinel-2, allow to map the cultivated fields and ongoing activities continuously throughout the harvesting period. 
 
-**Asparagus in Germany**
-Harvesting asparagus is a highly labour-intensive activity and involves seasonal workers who typically arrive from Bulgaria and Romania to support the German farmers during the harvest period from March to June.
-Sentinel-2 data allow an overview of the cultivated white asparagus fields and related activities.
+This indicator illustrates the ratio:
+
+\frac{Area_2019-Area_2020}{Area_2019}
```
Comment (lubojr) [inline collections/E10a3_agricultural_production_area_change/E10a3_agricultural_production_area_change.md:6]: @aapopescu Sorry, like this no. Although this is the way to do it in rich markdown that GitHub uses but for the StacInfo, we are working with the [commonmark](https://commonmark.org/help/), but we additionally allow some custom HTML, so I have tried following html way `<sup>Area_2019-Area_2020</sup>&frasl;<sub>Area_2019</sub>` and it seems to work but I am not sure if would be visually acceptable for you:
image

**Review by lubojr — APPROVED** [review]

File: collections/E10a3_agricultural_production_area_change/E10a3_agricultural_production_area_change.md (hunk 1 above)
Comment (lubojr) [reply collections/E10a3_agricultural_production_area_change/E10a3_agricultural_production_area_change.md]: Changed and merging

**Pull request merged**